utils/appliance/__init__.py

```diff
@@ -378,6 +378,7 @@ def _configure_5_3(self, log_callback=None):
     def _configure_upstream(self, log_callback=None):
         self.wait_for_evm_service(timeout=1200, log_callback=log_callback)
         self.deploy_merkyl(start=True, log_callback=log_callback)
+        self.add_ca_certs(log_callback=log_callback)
```

Comment:
Just a note, this will only happen for templates that get built after this gets merged. And please, notify me when this gets merged so I can update sprout to reflect this change ASAP.